docs/sources/mimir/configure/about-versioning.md

```diff
@@ -109,13 +109,15 @@ The following features are currently experimental:
     - `ingester.ring.token-generation-strategy`
     - `ingester.ring.spread-minimizing-zones`
     - `ingester.ring.spread-minimizing-join-ring-in-order`
+  - Allow ingester's `Push()` to return gRPC errors only: `-ingester.return-only-grpc-errors`.
 - Ingester client
   - Per-ingester circuit breaking based on requests timing out or hitting per-instance limits
     - `-ingester.client.circuit-breaker.enabled`
     - `-ingester.client.circuit-breaker.failure-threshold`
     - `-ingester.client.circuit-breaker.failure-execution-threshold`
     - `-ingester.client.circuit-breaker.period`
     - `-ingester.client.circuit-breaker.cooldown-period`
+  - Allow gRPC status codes to be used in `status_code` label of `cortex_ingester_client_request_duration_seconds` metric: `-ingester.client.report-grpc-codes-in-instrumentation-label-enabled`.
```

Comment:
This one has category advanced, not experimental.

Reply:
You are right, I will fix this.